specs/pvs/corla.pvs

```diff
@@ -470,6 +495,11 @@ ballot_manifests: THEORY
 BEGIN
   IMPORTING ballots, elections_equipment, information_systems
 
+% @todo kiniry Should this information also include what kind of
+% ballot each ballot is?  I.e., re-marked, phantom, etc.?
+% @todo kiniry These types will be refined when we review the
+% information provided by CDOS on 20 July 2017 about Dominion's file
+% formats.
   ballot_manifest_info: TYPE =
     [scanner_id_number, batch_number, batch_size, storage_container]
   ballot_manifest: TYPE = set[ballot_manifest_info]
```

Comment:
I've added `county_id` for the moment.  I'm not going to get ahead of ourselves with the design to include data model elements necessary for stratified audits.  We can/will put out a new version that will handle other audit styles, and of course the data model will evolve with those new releases.